rs/bitcoin/kyt/src/main.rs

```diff
@@ -127,8 +146,103 @@ fn post_upgrade(arg: KytArg) {
 #[ic_cdk::query(hidden = true)]
 fn http_request(req: http::HttpRequest) -> http::HttpResponse {
```

Comment:
1. We typically forbid update calls on that endpoint to avoid cycles draining attack, see [here](https://github.com/dfinity/ic/blob/15d752c5dd1247268e1355a0ca5efe655dc8cc75/rs/ethereum/ledger-suite-orchestrator/src/main.rs#L111) for example and the corresponding [test](https://github.com/dfinity/ic/blob/15d752c5dd1247268e1355a0ca5efe655dc8cc75/rs/ethereum/ledger-suite-orchestrator/tests/tests.rs#L294).
2. I would also add 2 smoke integrations tests (pocket IC) to ensure that metrics and logs can be retrieved, for example similar to [here](https://github.com/dfinity/ic/blob/15d752c5dd1247268e1355a0ca5efe655dc8cc75/rs/ethereum/ledger-suite-orchestrator/tests/tests.rs#L417)

Reply:
Thanks! Both are addressed.